<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Livelog</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Detekcja anomalii</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>Zakres danych</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Według kolejności:</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Od:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="spinBox">
                   <property name="maximum">
                    <number>99999</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>Do:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="spinBox_2">
                   <property name="maximum">
                    <number>99999</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_4">
                 <property name="text">
                  <string>Pobierz dane</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Według daty:</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Data początkowa:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDateTimeEdit" name="dateTimeEdit_2"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Data końcowa:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDateTimeEdit" name="dateTimeEdit"/>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_3">
                 <property name="text">
                  <string>Pobierz dane</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_6">
           <attribute name="title">
            <string>Parametry detekcji</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Metoda detekcji anomalii:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox">
                <item>
                 <property name="text">
                  <string>Metoda Losowa</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Sieć Kohonena - SOM</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Naiwny klasyfikator Bayesowski - Naive Bayes</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Metoda najbliższego sąsiada</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Metoda gęstościowa</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Sieć neuronowa radialna - RBF</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Sieć neuronowa zwykła - Neural Network</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Lokalny czynnik odbiegania - LOF</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>K najbliższych sąsiadów - KNN</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTabWidget" name="tabWidget_3">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tab_7">
               <attribute name="title">
                <string>SOM</string>
               </attribute>
              </widget>
              <widget class="QWidget" name="tab_8">
               <attribute name="title">
                <string>RBF</string>
               </attribute>
              </widget>
              <widget class="QWidget" name="tab_9">
               <attribute name="title">
                <string>Naive Bayes</string>
               </attribute>
              </widget>
              <widget class="QWidget" name="tab_10">
               <attribute name="title">
                <string>Neural Network</string>
               </attribute>
              </widget>
              <widget class="QWidget" name="tab_11">
               <attribute name="title">
                <string>LOF</string>
               </attribute>
              </widget>
              <widget class="QWidget" name="tab_12">
               <attribute name="title">
                <string>KNN</string>
               </attribute>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Od:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_4">
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Do:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_3">
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_7">
            <property name="text">
             <string>Naucz</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_8">
            <property name="text">
             <string>Testuj</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="LiveLineChart" name="webView">
            <property name="url">
             <url>
              <string>about:blank</string>
             </url>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_14">
       <attribute name="title">
        <string>Przegląd danych</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7"/>
      </widget>
      <widget class="QWidget" name="tab_13">
       <attribute name="title">
        <string>Raporty</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_9"/>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Subskrypcja</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6"/>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Ustawienia</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Badany obiekt</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Obecnie wybrany obiekt:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Obiekt 01 - Monitoring</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Obiekt:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_2">
               <item>
                <property name="text">
                 <string extracomment="18">Obiekt 01 - Monitoring</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string extracomment="22">Obiekt 02 - Monitoring</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string extracomment="28">Obiekt 03 - Monitoring</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string extracomment="9">Obiekt 04 - Monitoring</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string extracomment="28">Obiekt 05 - Monitoring</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string extracomment="22">Obiekt 06 - Monitoring</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string extracomment="16">Obiekt 07 - Monitoring</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string extracomment="15">Obiekt 08 - Monitoring</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string extracomment="28">Obiekt 09 - Monitoring</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string extracomment="28">Obiekt 10 - Monitoring</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string extracomment="22">Obiekt 11 - Monitoring</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string extracomment="22">Obiekt 12 - Monitoring</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string extracomment="28">Obiekt 13 - Monitoring</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string extracomment="33">Obiekt 14 - Monitoring</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string extracomment="16">Obiekt 15 - Monitoring</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string extracomment="16">Obiekt 16 - Monitoring</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string extracomment="18">Obiekt 17 - Monitoring</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_5">
               <property name="text">
                <string>Wybierz</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Połączenie z bazą danych</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Nazwa komputera:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEdit_3"/>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEdit_4">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Hasło:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Użytkownik:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Baza danych:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QPushButton" name="pushButton_6">
             <property name="text">
              <string>Połącz</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
  <customwidget>
   <class>LiveLineChart</class>
   <extends>QWebView</extends>
   <header>GUI_COMPONENTS/livelinechart.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>chooseObject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>connectDatabase()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>558</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadDataDate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadDataStandard()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>learnData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>testData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>chooseObject()</slot>
  <slot>connectDatabase()</slot>
  <slot>loadDataStandard()</slot>
  <slot>loadDataDate()</slot>
  <slot>learnData()</slot>
  <slot>testData()</slot>
 </slots>
</ui>
